src/ReadVTK.jl

```diff
@@ -126,16 +128,27 @@ function VTKFile(filename)
   # Ensure supported compression type
   @assert in(compressor, ("", "vtkZLibDataCompressor"))
 
-  # Extract piece
-  piece = root[file_type][1]["Piece"][1]
-  n_points = parse(Int, attribute(piece, "NumberOfPoints", required=true))
-  n_cells = parse(Int, attribute(piece, "NumberOfCells", required=true))
+  if file_type == "UnstructuredGrid"
+    # Extract piece
+    piece = root[file_type][1]["Piece"][1]
+    n_points = parse(Int, attribute(piece, "NumberOfPoints", required=true))
+    n_cells = parse(Int, attribute(piece, "NumberOfCells", required=true))
+
+  elseif file_type == "ImageData"
+    piece = root[file_type][1]
+    wholeExtent = parse.(Int,     split( attribute(piece, "WholeExtent", required=true) , ' ' ) )
```

Comment:
What you are reading in here is not a "piece" in VTK parlance, but a "dataset element", thus I suggest to change it to
```suggestion
    dataset_element = root[file_type][1]
    wholeExtent = parse.(Int, split(attribute(dataset_element, "WholeExtent", required=true), ' '))
```
I also removed unnecessary whitespace

Reply:
excellent. thanks.